docs/concepts/how-ipfs-works.md

```diff
@@ -5,86 +5,168 @@ description: Learn how the InterPlanetary File System (IPFS) works and why it's
 
-There are [other content replication protocols under discussion](https://github.com/ipfs/camp/blob/master/DEEP_DIVES/24-replication-protocol.md) as well, the most developed of which is [_Graphsync_](https://github.com/ipld/ipld/blob/master/specs/transport/graphsync/index.md). There's also a proposal under discussion to [extend the Bitswap protocol](https://github.com/ipfs/go-bitswap/issues/186) to add functionality around requests and responses.
+- [Bitswap](#bitswap-for-data-transfer)
+- [IPFS HTTP Gateways](#ipfs-http-gateways)
+- [Sneakernet](#sneakernet)
 
-## SHA file hashes won't match Content IDs
+### Bitswap (for data transfer)
 
-You may be used to verifying the integrity of a file by matching SHA hashes, but SHA hashes won't match CIDs. Because IPFS splits a file into blocks, each block has its own CID, including separate CIDs for any parent nodes.
+As discussed in [How content routing works in IPFS](#bitswap-for-content-routing), IPFS nodes use Bitswap, a <VueCustomTooltip label="Unlike a request-response protocol, all nodes in the system receive every message transmitted, and decide whether the message received should be immediately discarded, stored or processed." underlined multiline is-medium>message-based</VueCustomTooltip>,  <VueCustomTooltip label="A network of computers model in which each party has equivalent capabilities and can initiate a communication session." underlined multiline is-medium>peer-to-peer network</VueCustomTooltip> protocol for both content routing and the transfer of data. With Bitswap, any peers that an IPFS node is connected to can transfer requested blocks directly to that node without needing to traverse the [DHT](#kademlia-distributed-hash-table-dht). Peers also store wantlists, so that if a peer receives requested data at a later time, it can then transfer it to the node that originally requested.
 
-The DAG keeps track of all the content stored in IPFS as blocks, not files, and Merkle DAGs are self-verified structures. To learn more about DAGs, see [directed acyclic graph (DAG)](../concepts/merkle-dag.md).
+:::callout
+**Learn more**
 
-For a detailed example of what happens when you try to compare SHA hashes with CIDs, see [Content Identifiers are not hashes](../concepts/hashing.md#content-identifiers-are-not-file-hashes).
+Want learn more about Bitswap? See the [Bitswap deep dive](../concepts/bitswap.md).
+:::
+
+### IPFS HTTP Gateways
+
+HTTP Gateways allow applications that do not support or implement all IPFS subsystems to fetch data from the IPFS network using an HTTP interface. In its simplest form, a gateway is an IPFS Node that also exposes an [HTTP Gateway API](https://github.com/ipfs/specs/blob/main/http-gateways/README.md).
+
+:::callout
+**Learn more**
+
+Want learn more about IPFS Gateways? See the [IPFS Gateway conceptual guide](../concepts/ipfs-gateway.md).
+:::
 
-### Libp2p
+### Sneakernet
```

Comment:
This section is unnecessarily long and goes too much into the reasons **why** you would use sneakernet. 

The main idea to get across here is that you can move CAR files using external drives without any network connectivity and still get the benefits of IPFS.